test/interop/C/llvm/makefiles/dynamicLinking/PRECOMP

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-chpl --llvm --library --dynamic FooLibrary.chpl
+$3 --llvm --library --dynamic FooLibrary.chpl
```

Comment:
I'm not objecting but I don't see what this change has to do with pic?

Reply:
This was a test that failed when I ran because it could not find `chpl` (have to use $3 for absolute path). I figured I'd just fix it now.

Reply:
My commit message might be misleading but these failed during the `pic` run.